src/commands/run.ts

```
@@ -2,6 +2,11 @@ import { RunOptions } from "../types";
 import { getLogger } from "../utils";
 import { DBService, ApiService, ChainContext } from "../services";
 
+// How often to log the process memory usage. This leaves a memory trail in the
+// logs so a restart can be correlated with memory pressure (e.g. OOM) even when
+// only the application logs are available.
+const MEMORY_LOG_INTERVAL_MS = 60_000;
```

ESLint

[error] 'MEMORY_LOG_INTERVAL_MS' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)